bin/cli.py

```diff
@@ -19,98 +19,40 @@ def train(
     name: str = typer.Option(
         ..., "-n", "--name", help="Name of experiment to save under."
     ),
+    epochs: int = typer.Option(
+        2, "-e", "--epoch", help="Number of epochs."
+    ),
+    batch_size: int = typer.Option(
+        1000, "-bs", "--batch_size", help="Batch size."
+    ),
+    learning_rate: float = typer.Option(
+        0.01, "-lr", "--learning_rate", help="Learning rate."
+    ),
 ):
     print(f"Running experiment {name}")
     device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-    epochs = 2
-    batch_size = 1000
-    learning_rate = 0.01
 
     # save the parameters!
+    ####################### need to do this #############

```

Comment:
if it works you can remove this comment. Always try to keep your documentation clean!